db/seeds.rb

```diff
@@ -5,3 +5,42 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+
+Answer.delete_all
+ApplicationRecord.delete_all
+Keyword.delete_all
+Officer.delete_all
+Question.delete_all
+Rso.delete_all
+
+rso.create!(name: 'Association for Computer Machinery',
```

Comment:
@seattlecoder9 why are you using exclamation points on all of these? Also can we capitalize all of the class names?